utils/stop_all_motors.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#!/usr/bin/env micropython
```

Comment:
I don't think we want to change this globally...

Reply:
It stops the motors so much faster though...what's the downside?

Reply:
Mhmmm... I guess that makes sense. The code's all the same so people can still refer to it, so I don't see any real harm.